### ISSUE TITLE
Tags seem not to be indexed

### ISSUE THREAD

**Issue by trugwaldsaenger** [hbz/oerworldmap#1729]
When I click on the tag "esd - education for sustainable development" I receive 23 hits:
https://oerworldmap.org/resource/?filter.about.keywords=esd%20-%20education%20for%20sustainable%20development%20&size=20&features=true
When I search for "esd - education for sustainable development" I receive 0 hits. 

Does this mean that our tags are not yet indexed/searchable? I thought we solved this issue in the past, but cannot find it right now on Git...

**Comment by literarymachine** [comment]
> When I search for "esd - education for sustainable development" I receive 0 hits.
> Does this mean that our tags are not yet indexed/searchable?

No, in general tags are indexed and thus searchable, see e.g. https://oerworldmap.org/resource/?q=auninquekeyword&size=20

It seems to me like the dash is causing trouble here, thus the scope of the problem is not huge.
